content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -76,3 +76,195 @@ you can view processes in other containers.
 
 * Learn how to [debug pods using ephemeral containers](/docs/tasks/debug/debug-application/debug-running-pod/#ephemeral-container).
 
+### Kubernetes Ephemeral Container Security
+Attempting to debug a Pod and realizing that you can't install curl due to security settings has to be a meme at this point. Good security practices are always nice but it often comes at the cost of usability. To the point where some may even solve this problem by installing debug tools into their production images. Shudders.
```

Comment:
Please manually wrap long lines like this.

Long lines are difficult to comment, to translate, to track changes.